packages/@ember/-internals/container/lib/container.ts

```diff
@@ -2,7 +2,7 @@ import { Factory, LookupOptions, Owner, OWNER, setOwner } from '@ember/-internal
 import { dictionary, HAS_NATIVE_PROXY } from '@ember/-internals/utils';
 import { EMBER_MODULE_UNIFICATION } from '@ember/canary-features';
 import { assert } from '@ember/debug';
-import { assign } from '@ember/polyfills';
+import { assign, WeakSet } from '@ember/polyfills';
```

Comment:
Hmm, why is `WeakSet` added here (it wasn't imported previously)?

Reply:
Isn't it used here https://github.com/emberjs/ember.js/pull/16990/files/04fbecca7d3f0a731f653bcc0023f3d3d8d782d9#diff-ed11ef2c87b370bc81065deecede9698L33